### ISSUE TITLE
LC is replcing Unicode characters in .SRT files with gibberish

### ISSUE THREAD

**Issue by bigfella237** [mifi/lossless-cut#973]
An example from an .SRT subtitle file before being Lossless Cut: 

https://i.postimg.cc/4xdMFHzs/Subs-pre-Lossless-Cut.jpg

And after:

https://i.postimg.cc/jqHGWbx0/Subs-post-Lossless-Cut.jpg

As you can see, Lossless Cut is replacing the Unicode "music note" character ♪ with "â™ª". I assume it may be replacing other Unicode characters as well but I've only seen the music note so far.

This creates a lot of work (especially with multiple subtitle tracks) as I need to extract the tracks from the .MKV output file, open and edit them, then mux everything back together again.

Could Lossless Cut be made to output the same Unicode characters as were in the input file?

**Comment by mifi** [comment]
This must be a bug in ffmpeg. losslesscut doesn't do any processing of subtitle tracks. Could you provide a reproduceable simple test case with an ffmpeg command only? (you can find recently run ffmpeg commands in the help panel of losslesscut)

If we have a reproduceable ffmpeg-only test case, then we can report this to ffmpeg
https://ffmpeg.org/bugreports.html

**Comment by mifi** [comment]
Perhaps try with ffmpeg's built in ffmpeg version first and then with the newest version of ffmpeg and see if they have fixed it.

**Comment by bigfella237** [comment]
Sorry but I don't know anything about ffmpeg, I don't think I've ever used it.

I simple open Lossless Cut, drag 'n drop an .MKV file into it, specify the start and end cut points and export.

And I have since confirmed that it corrupts all Unicode characters within an .SRT file, not just the music note.

**Comment by mifi** [comment]
If you could provide a sample video file or srt file that can reproduce this problem, then I can test it with ffmpeg for you

**Comment by bigfella237** [comment]
Apologies for the late reply, I've been trying to figure out what's going on...

I made a video clip with my own subtitle track to demonstrate the problem but when I trimmed it with Lossless Cut it didn't do it. So back-tracking the issue I've found that it relates only to Netflix .srt files, there's something hinky with the way they encode their files (not really surprising for them).

If I open their subs in my editor and save them again it doesn't happen so I can only assume that Lossless Cut isn't at fault.

Sorry and thanks anyway!
